yarn.lock

```diff
@@ -1883,7 +1883,7 @@
     "@floating-ui/core" "^1.4.1"
     "@floating-ui/utils" "^0.1.1"
 
-"@floating-ui/react-dom@^2.0.2":
+"@floating-ui/react-dom@^2.0.1", "@floating-ui/react-dom@^2.0.2":
```

Comment:
Strange 🤔. My gut feeling tells me that it should have just the latest version here.

Reply:
Do you know what could be causing this?

Reply:
Found the issue 😅 

- https://github.com/mui/material-ui/pull/38940
- https://github.com/mui/material-ui/pull/38951